packages/vue/src/index.ts

```diff
@@ -243,10 +243,10 @@ const Frontegg: PluginObject<PluginOptions> | any = {
         this.fronteggAuth = Vue.fronteggAuth;
         this.loginWithRedirect = loginWithRedirect.bind(this);
 
-        // _entitlements was added for to make the computed property reactive, then it will get updated
-        this.getFeatureEntitlements = (_entitlements: User['entitlements'], key: string) => fronteggApp.getFeatureEntitlements(key);
-        this.getPermissionEntitlements = (_entitlements: User['entitlements'], key: string) => fronteggApp.getPermissionEntitlements(key);
-        this.getEntitlements = (_entitlements: User['entitlements'], entitledToOptions: EntitledToOptions) => fronteggApp.getEntitlements(entitledToOptions);
+        // _user was added for to make the computed property reactive, then it will get updated
+        this.getFeatureEntitlements = (_user: User | undefined | null, key: string, customAttributes?: CustomAttributes) => fronteggApp.getFeatureEntitlements(key, customAttributes);
```

Comment:
THIS IS A BREAKING CHANGE!
2 options:
- Mark it as a major
- Talk to customers

DONT MERGE BEFORE